### ISSUE TITLE
There should be a way to see if an uploaded file was uploaded by a registered user or not

### ISSUE THREAD

**Issue by dongmaster** [Volafile/volafile-bugs#143]
There has been a problem in the big room with 600 people in it where someone would change their name to the name of a registered user while not logged in and that person would then upload a request under another persons name.

You could add a field in the tool tip that appears when you hover over a file that says:
Registered: Yes
or Registered: No
or you could make the name tag a special colour for when a user is logged in while having uploaded the file.

**Comment by bastardsandcheats** [comment]
Would be better adding a character to the username so when you do search files for user:username you could search just for the registered user
 so registered users would be - username^ 
and regular users would be - username

**Comment by Heisenb3rgx** [comment]
I ran across this issue today myself and think it would be beneficial to have a system that allows for username status colours in the filebin. If classes were added per level (white, green, light green, purple) then a script or stylish theme could be created so that the default look of vola wasn't changed.

This would not only prevent impersonation, but also allow for certain rooms to be kept member-only by easily spotting and removing anything uploaded anonymously.
